.tsconfig

```diff
@@ -0,0 +1,35 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "target": "es6",
+    "lib": ["dom", "dom.iterable", "es6"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "noEmit": false,
+    "jsx": "react",
+    "sourceMap": true,
+    "isolatedModules": true,
+    "downlevelIteration": false,
+    "rootDir": "./",
+    "baseUrl": "src/",
+    "watch": true,
+    "noUnusedLocals": true,
+    "types": ["react", "node", "jest"],
+    "paths": {
+      "@components/*": ["components/*"],
+      "@assets/*": ["assets/*"],
+      "@layouts/*": ["layouts/*"],
+      "@pages/*": ["pages/*"],
+      "@utils/*": ["utils/*"]
+    },
+    "pretty": true
+  },
+  "esclude": ["node_modules"]
+}
```

Comment:
**suggestion** Shall we add a newline to the end of the file? That's usually a default.

Reply:
I don't use to add a newline at the end, but we can enable by default on eslint if I am not wrong.

Reply:
Let's do it, I think it's a practice coming from the past. https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file :)

Reply:
it should work now, I added rule: "newLine": "LF", and added a empty line at the end of the file. Is it what we do?

Reply:
Yup! https://github.com/stormkit-io/app-stormkit-io/pull/18 seems to be similar to this branch. Shall we close this PR or they are different?

Reply:
same branch but different PR